packages/next/client/components/headers.ts

```diff
@@ -17,7 +17,7 @@ function staticGenerationBailout(reason: string) {
   }
 }
 
-export function headers(): Headers {
+export function headers() {
```

Comment:
Typed in `packages/next/client/components/request-async-storage.ts` already, similar to `cookies()`